src/welldev/io/controller/UpdatePasswerd.java

```diff
@@ -0,0 +1,39 @@
+package welldev.io.controller;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import welldev.io.DAO.Driver;
+
+@WebServlet("/updatepasswerd")
+public class UpdatePasswerd extends HttpServlet {
+	
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		HttpSession session = request.getSession();
+		String un = (String) session.getAttribute("un");
+		String p = (String) session.getAttribute("p");
+		
+		if(p!=null) {
```

Comment:
same goes here.. you can add filter to check session.. rather checking in every file